index.html

```diff
@@ -0,0 +1,45 @@
+<<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <section id="main" class="main">
+      <h1>Dr. Norman Borlaug</h1>
+      <h5>The man who saved a billion lives</h5>
+
+      <a href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="blank" class="project project-tile">
+
+        <img class="project-image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="project">
+</a>
```

Comment:
[optional] try to have some alignment in opening and closing tags to help you have some bearing on the positioning of things